dxr/config.py

```diff
@@ -105,7 +105,7 @@ def __init__(self, input, relative_to=None):
                     AbsPath,
                 Optional('workers', default=if_raises(NotImplementedError,
                                                       cpu_count,
-                                                      1)):
+                                                      1) / 2):
```

Comment:
That's going to cause interesting behavior when cpu_count() is 1 (like on the default vagrant VM), since 1 / 2 == 0. Also, VMs assigned <= the physical number of CPUs on the host will be unjustly penalized by this. We could start using https://pypi.python.org/pypi/psutil/, which can report the number of physical cores, but that's a big dependency with a platform-specific component and a compilation step for a fairly small convenience. I'm also not sure that setting `workers` to include the logical cores isn't a help during clang indexing, when some processes sleep while sending data to ES. (I notice my VM is currently set to physical+logical cores, presumably for a reason.) Do you think a note in the documentation would suffice? Do you think depending on a 200K library is worth it? (This may be clearer to me after lunch.)